ddtrace/tracer.py

```diff
@@ -81,6 +82,13 @@
 AnyCallable = TypeVar("AnyCallable", bound=Callable)
 
 
+# We have to reseed the RNG or we will get collisions between the processes as
```

Comment:
Should we just do this in the `_rand` module?

Reply:
It would be nice and tidy, but how do we ensure that the hook gets registered? It would be left to at least a loaded module to load `_rand`.

Reply:
Oh right. Would it make sense at the module-level of `_rand`? (When the `_rand` module is imported, the hook is registered?)

Reply:
Yes that's what I'm saying 🙂 , but now that I think it through, we'd only care to register the hook only if the module is imported, otherwise we don't need the hook.

Reply:
Right - it's only needed if the module is imported 😄